master/de.po

```diff
@@ -29907,9 +29935,8 @@ msgid "Center on horizontal and vertical axis"
 msgstr "An horizontalen und vertikalen Achsen ausrichten"
 
 #: ../src/widgets/arc-toolbar.cpp:140
-#, fuzzy
 msgid "Change arc"
-msgstr "Weichzeichner ändern"
+msgstr "Bogen ändern"
```

Comment:
Ich würde "Kreisbogen" oder "Kreissegment" vorschlagen (mit Tendenz zum "Kreisbogen").

Während "arc" im Englischen relativ eindeutig ist, is ein "Bogen" im Deutschen relativ unspezifisch und kann eine ganze Menge sein.

Reply:
Jo, wollte ich auch erst - habe ich dann aber nicht gemacht, weil es um die (unterschiedlich großen) Radien für Ellipsen geht. Ist auch nur die Nachricht in der Undo-History. Ellipsenbogen fand ich dann wieder eher unpassend. Hast Du noch eine andere Idee?

Reply:
Oh, und: Danke für's Drüberschauen! :-D

Reply:
Achso, mein Vorschlag war allgemein auf "arcs" bezogen, nicht nur auf diese konkrete Vorkommen (etwa unten "Horizontaler Radius des Kreises, der Ellipse oder des **Kreis**bogens")

An dieser Stelle ist tatsächlich schon das englische Original etwas unglücklich gewählt, denn die Meldung erscheint für Kreise, Ellipsen und (Kreis)bögen" - Wenn wir davon nicht abweichen wollen finde ich trotzdem "Kreisbogen" am besten (passt auf Kreise und Ellipsen ebenfalls - wenn auch nicht ganz eindeutig) - idealerweise würde da natürlich irgendwas mit Radius auftauchen (ohne sich auf die Form zu beziehen).

Reply:
'Change circle/ellipse radius' also für inkscape master? Wir haben zwar keinen string freeze, und das ist nur eine Undo-Nachricht, aber für 0.92.3 ist das vielleicht dann doch nicht so nett.

Reply:
Auf jeden Fall besser als nur "arc" - bin immer noch am überlegen ob es einen  guten (und allgemeinverständlichen) "Sammelbegriff" für "cirvle/ellipse/arc" gibt.

Ich habe eben auch ein paar der Meldungen angeschaut die da so auftauchen und es scheint generell ziemlich inkonsistent zu sein (könnte man vielleicht auch mal vereinheitlichen).